manifests/f/FireDaemon/OpenSSL/3.1.3/FireDaemon.OpenSSL.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: FireDaemon.OpenSSL
+PackageVersion: 3.1.3
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download.firedaemon.com/FireDaemon-OpenSSL/FireDaemon-OpenSSL-x64-3.1.3.exe
+  InstallerSha256: C897942F2F319E070DCD0D882149E40B700ACAAA27532CC31439E172B5840DF1
+  InstallModes:
+  - silent
+  InstallerSwitches:
+    Silent: /exenoui /exelog fdopenssl3.log /qn /norestart REBOOT=ReallySuppress

```

The log parameter should be moved into the `Log` installer switch
```suggestion
    Silent: /exenoui /qn
    Log: /exelog <LOGPATH>
    Custom: /norestart REBOOT=ReallySuppress
```